docs/design/index.md

```
@@ -165,12 +165,13 @@ Queue: Hiring Queue
 
 <div class="grid cards" markdown>
 
--   [**Agents & HR**](agents.md)
+-   [**Agents**](agents.md) / [**HR & Agent Lifecycle**](hr-lifecycle.md)
```

markdownlint-cli2 (0.22.0)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)